index.html

```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
+        <div>
+          <h4>↓ Work</h4>
+        </div>
+        <div>
+          <h2>MY RECENT <br> WORK</h2>
+        </div>
+      </section>
+      <section class="grid projects">
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
+        </article>
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
+        </article>
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
+        </article>
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
```

Comment:
- [ ] One of the requirements for this project is that. the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should all be a list `ul` and each of them a `li`. This helps you to write semantic code which is a good practice. Please correct this for all other project sections in your portfolio

Reply:
Well resolved